CIP-ledger-evolution/README.md

```diff
@@ -0,0 +1,302 @@
+
+To become active, the ledger era MUST:
+* specify the semantics with the [STS](STS.md) framework
+* specify cryptographic details in an appendix to the STS specification
+* specify the CBOR serialization schema with a CDDL file
+* be included in the software update for a soft/hard fork.
+
+### Serialization changes
+
+See [Serialization](Serialization.md).
+
+### The Small Step Semantics Framework for Cardano
+
+See [STS](STS.md).
+
+### Implementing and releasing changes
+
+This CIP does not cover the process of implementing changes.
+As usual, the CIP process covers the design phase, and it is up to the implementer to ensure that
+their proposal is implemented, which may require additional work to meet the requirements of the
+maintainers of the Cardano code repositories (testing, implementation quality, approach), and so on.
+
+Changes can be released after their CIPs have reached Active status.
+Different changes will require different releases as described in "Types of release".
+This CIP does not cover the process by which changes are actually incorporated into releases after having been implemented.
+In particular, there is NO assumption that a feature which requires a particular release will be
+included in the next such release, even after it has been implemented.
+
+### Ledger Core CIP registry
+
+Any CIP which proposes a type of change listed in "Types of change" MUST also add itself to this registry (in addition to the main registry).
+
+| #  | Title             | Type of change          | Status |
+|----|-------------------|-------------------------|--------|
+| 31 | Reference inputs  | Ledger-script interface | Draft  |
+| 32 | Inline datums     | Ledger-script interface | Draft  |
+| 33 | Reference scripts | Ledger-script interface | Draft  |
+
+## Rationale
+
+### Why have a public process for changes?
+
+Cardano is continuing to move towards decentralized governance within the Voltaire phase of development.
+Historically, key development and implementation decisions have been made by the core development team.
+This was important in the earliest stages of the platform’s evolution.
+However, this becomes less so as the platform starts to mature and is neither sustainable nor desirable in the long term.
+
+Furthermore, while many changes to Cardano are obscure or not of interest to many community members,
+there is a much larger community who have a keen interest in changes to Plutus Core: dApp developers.
+Hence it is especially important to have a clear way for this community to be able to propose changes and see how they are progressing.
+
+### Why include a CIP registry?
+
+This is just to make it easy for those considering proposing a CIP following this process to see which CIPs have already been submitted.
+An alternative would be a standard title for CIPs, or perhaps some kind of CIP metadata to indicate that it follows the process in this CIP.
+
+### Out of scope
```

Comment:
Should this be in "Rationale"? Also this seems to cover the same ground as "Changes that do not require a CIP"